includes/Type/InterfaceType/EditorBlockInterface.php

```diff
@@ -21,12 +20,20 @@ final class EditorBlockInterface {
 	/**
 	 * Gets the block from the Block Registry.
 	 *
-	 * @param array      $block   The block being resolved.
-	 * @param AppContext $context The AppContext.
+	 * @param array $block The block being resolved.
+	 * @param null  $context Deprecated.
 	 *
 	 * @return \WP_Block_Type|null
 	 */
-	public static function get_block( array $block, AppContext $context ) {
+	public static function get_block( array $block, #[Deprecated] $context = null ) {
```

Comment:
Isn't `#[Deprecated]` a PHP 8 attribute? If so, the plugin header says we support 7.2.

Reply:
[I stand corrected](https://php.watch/versions/8.0/attributes#:~:text=The%20final%20syntax%20of%20Attributes,run%20on%20earlier%20PHP%20versions.), what I said above is only true if we make the params multiline.

Your call whether to break it multiline or just drop the 8.0 hinting for something that probably wont stay in the codebase for that long 🤷